.coveragerc

```diff
@@ -25,5 +25,5 @@ exclude_lines =
     .*:.* # Python \d.*
     .* # Abstract
     .* # Defensive:
-    if TYPE_CHECKING:
+    if (?:typing.)?TYPE_CHECKING:
```

Comment:
nit: I would prefer using the same form throughout the project

Reply:
Agreed, I've created this issue: https://github.com/urllib3/urllib3/issues/2789